UnitsNet.Tests/CustomCode/LengthTests.cs

```diff
@@ -233,5 +233,24 @@ public static void InverseReturnsReciprocalLength(double value, double expected)
 
             Assert.Equal(expected, inverseLength.InverseMeters);
         }
+
+        [Theory]
+        [InlineData(3, 2.563, 16, "3' - 2 9/16\"")]
+        [InlineData(3, 2.563, 32, "3' - 2 9/16\"")]
+        [InlineData(3, 2, 16, "3' - 2\"")]
+        [InlineData(3, 2.5, 1, "3' - 2\"")]
+        [InlineData(0, 2, 32, "2\"")]
```

Comment:
Nice tests.

Can we also add these test values from my example in another PR comment? They helped me understand how it works better. 

Adapt the example to [InlineData].

```
var length = Length.FromFeetInches(3, 2.6);
length.ToArchitecturalString(1)   => 3' - 3"
length.ToArchitecturalString(2)   => 3' - 2 1/2"
length.ToArchitecturalString(4)   => 3' - 2 1/2"
length.ToArchitecturalString(8)   => 3' - 2 5/8"
length.ToArchitecturalString(16)  => 3' - 2 5/8"
length.ToArchitecturalString(32)  => 3' - 2 19/32"
length.ToArchitecturalString(128) => 3' - 2 77/128"
```